http_request_randomizer/requests/useragent/userAgent.py

```diff
@@ -1,15 +1,25 @@
 import os
 import random
+from fake_useragent import FakeUserAgent
+import logging
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
 
 class UserAgentManager:
-    def __init__(self, agent_file=os.path.join(os.path.dirname(__file__), '../data/user_agents.txt')):
-        self.agent_file = agent_file
-        self.useragents = self.load_user_agents(self.agent_file)
+    def __init__(self, fallback=None, file=None):
```

Comment:
The user can specify an arbitrary user agent as a fallback. It would be used in the rare occasion when some error happens in fake-useragent while processing the request (network error etc)

Reply:
Hey @la55u 
Just took a better look at the library, my suggestions:
* Let's avoid storing collected data at the os temp dir (keep everything in memory) by using `cache=False` argument
* Also lets always use a fallback browser (could be the most popular one) as we dont want out library to fail for any reason